core/modules/modscrumboard.class.php

```diff
@@ -53,7 +53,7 @@ public function __construct($db)
 
 		// Family can be 'crm','financial','hr','projects','products','ecm','technic','other'
 		// It is used to group modules in module setup page
-		$this->family = "projects";
+		$this->family = 'ATM Consulting - BTP';
```

Comment:
pour le kanban c'était pas mal projects ...
```suggestion
		$this->family = 'projects';
```